diff --git a/modules/custom-redirection/custom-redirection.php b/modules/custom-redirection/custom-redirection.php
index 3604ed3..a1cbbff 100644
--- a/modules/custom-redirection/custom-redirection.php
+++ b/modules/custom-redirection/custom-redirection.php
@@ -172,10 +172,10 @@ public function get_redirect_for_user( $user, $type = 'login', $default = '' ) {
 	 * @access public
 	 */
 	public function login_form() {
-		if ( ! empty( $_REQUEST['redirect_to'] ) ) {
-			$referer = wp_unslash( $_REQUEST['redirect_to'] );
-		} elseif ( wp_get_original_referer() ) {
+		if ( wp_get_original_referer() ) {
 			$referer = wp_get_original_referer();
+		} elseif ( ! empty( $_REQUEST['redirect_to'] ) ) {
+			$referer = wp_unslash( $_REQUEST['redirect_to'] );
 		} else {
 			$referer = Theme_My_Login::is_tml_page() ? wp_get_referer() : wp_unslash( $_SERVER['REQUEST_URI'] );
 		}
