diff --git a/modules/custom-redirection/custom-redirection.php b/modules/custom-redirection/custom-redirection.php
index 3604ed3..4b56918 100644
--- a/modules/custom-redirection/custom-redirection.php
+++ b/modules/custom-redirection/custom-redirection.php
@@ -106,6 +106,9 @@ public function get_redirect_for_user( $user, $type = 'login', $default = '' ) {
 		// Make sure the user has a role
 		if ( is_multisite() && empty( $user->roles ) ) {
 			$user->roles = array( 'subscriber' );
+			if ( is_super_admin() ) {
+				$user->roles = array( 'administrator' );
+			}
 		}
 
 		// Get the user's role
diff --git a/modules/themed-profiles/themed-profiles.php b/modules/themed-profiles/themed-profiles.php
index c058f3a..de01f34 100644
--- a/modules/themed-profiles/themed-profiles.php
+++ b/modules/themed-profiles/themed-profiles.php
@@ -117,6 +117,10 @@ public function init() {
 			if ( is_multisite() && empty( $user_role ) )
 				$user_role = 'subscriber';
 
+			if ( is_super_admin() ) {
+			    $user_role = 'administrator';
+			}
+
 			if ( 'profile.php' == $pagenow && ! isset( $_REQUEST['page'] ) ) {
 				if ( $this->get_option( array( $user_role, 'theme_profile' ) ) ) {
 					if ( ! empty( $_GET ) )
@@ -189,6 +193,10 @@ public function show_admin_bar( $show_admin_bar ) {
 		if ( is_multisite() && empty( $user_role ) )
 			$user_role = 'subscriber';
 
+		if ( is_super_admin() ) {
+			$user_role = 'administrator';
+		}
+
 		if ( $this->get_option( array( $user_role, 'restrict_admin' ) ) )
 			return false;
 
@@ -300,6 +308,10 @@ public function tml_display_profile( &$template ) {
 		if ( is_multisite() && empty( $user_role ) )
 			$user_role = 'subscriber';
 
+		if ( is_super_admin() ) {
+			$user_role = 'administrator';
+	        }
+
 		$_template = array();
 
 		// Allow template override via shortcode or template tag args
@@ -342,6 +354,10 @@ public function site_url( $url, $path, $orig_scheme = '' ) {
 			if ( is_multisite() && empty( $user_role ) )
 				$user_role = 'subscriber';
 
+			if ( is_super_admin() ) {
+				$user_role = 'administrator';
+			}
+
 			if ( $user_role && ! $this->get_option( array( $user_role, 'theme_profile' ) ) )
 				return $url;
 
